ibis/backends/pyspark/__init__.py

```diff
@@ -276,7 +276,9 @@ def table(self, name: str, database: str | None = None) -> ir.Table:
         qualified_name = self._fully_qualified_name(name, database)
 
         schema = self.get_schema(qualified_name)
-        node = ops.DatabaseTable(name, schema, self, namespace=database)
+        node = ops.DatabaseTable(
```

Comment:
Can this codecov warning possibly be correct?  I'm struggling to imagine us not grabbing a table in the `pyspark` tests.

Reply:
Pretty sure this is incorrect. I suspect clobbering of coverage information.